util/county_to_dcid.py

```diff
@@ -0,0 +1,3387 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""State Alpha2 to County to dcid."""
+COUNTY_MAP: dict = {
+  "AL": {
+    "Autauga County": "geoId/01001",
+    "Baldwin County": "geoId/01003",
+    "Barbour County": "geoId/01005",
+    "Bibb County": "geoId/01007",
+    "Blount County": "geoId/01009",
+    "Bullock County": "geoId/01011",
+    "Butler County": "geoId/01013",
+    "Calhoun County": "geoId/01015",
+    "Chambers County": "geoId/01017",
+    "Cherokee County": "geoId/01019",
+    "Chilton County": "geoId/01021",
+    "Choctaw County": "geoId/01023",
+    "Clarke County": "geoId/01025",
+    "Clay County": "geoId/01027",
+    "Cleburne County": "geoId/01029",
+    "Coffee County": "geoId/01031",
+    "Colbert County": "geoId/01033",
+    "Conecuh County": "geoId/01035",
+    "Coosa County": "geoId/01037",
+    "Covington County": "geoId/01039",
+    "Crenshaw County": "geoId/01041",
+    "Cullman County": "geoId/01043",
+    "Dale County": "geoId/01045",
+    "Dallas County": "geoId/01047",
+    "De Kalb County": "geoId/01049",
+    "DeKalb County": "geoId/01049",
+    "Elmore County": "geoId/01051",
+    "Escambia County": "geoId/01053",
+    "Etowah County": "geoId/01055",
+    "Fayette County": "geoId/01057",
+    "Franklin County": "geoId/01059",
+    "Geneva County": "geoId/01061",
+    "Greene County": "geoId/01063",
+    "Hale County": "geoId/01065",
+    "Henry County": "geoId/01067",
+    "Houston County": "geoId/01069",
+    "Jackson County": "geoId/01071",
+    "Jefferson County": "geoId/01073",
+    "Lamar County": "geoId/01075",
+    "Lauderdale County": "geoId/01077",
+    "Lawrence County": "geoId/01079",
+    "Lee County": "geoId/01081",
+    "Limestone County": "geoId/01083",
+    "Lowndes County": "geoId/01085",
+    "Macon County": "geoId/01087",
+    "Madison County": "geoId/01089",
+    "Marengo County": "geoId/01091",
+    "Marion County": "geoId/01093",
+    "Marshall County": "geoId/01095",
+    "Mobile County": "geoId/01097",
+    "Monroe County": "geoId/01099",
+    "Montgomery County": "geoId/01101",
+    "Morgan County": "geoId/01103",
+    "Perry County": "geoId/01105",
+    "Pickens County": "geoId/01107",
+    "Pike County": "geoId/01109",
+    "Randolph County": "geoId/01111",
+    "Russell County": "geoId/01113",
+    "St. Clair County": "geoId/01115",
```

Comment:
This seems like super tailored to your dataset. How did you build these maps?

Reply:
I added a description to the README:



The geoIds of the U.S Counties was generated using the County's real FIPS code from:
https://www.nrcs.usda.gov/wps/portal/nrcs/detail/national/home/?cid=nrcs143_013697

For example, Miami-Dade County would be geoId/12086 because Miami-Dade's FIPS code is 12086.
The USDA website above lists only the unique part of the name, that is "Miami-Dade" instead of "Miami-Dade County".

The words ["County", "Parish", "Borough"] were appended to all names except those that were Cities, Parishes or Boroughs.

Some counties have two different keys that map to the same value/geoId, though.
For example:

{
    "De Kalb County": "geoId/17037",
    "DeKalb County": "geoId/17037"
}

The reason for this is because the Google Mobility dataset uses an alternative name to refer to the same region.
Any modifications to the names were duplicated.